reference/configuration_files/client_certificates.rst

```diff
@@ -0,0 +1,14 @@
+.. _client_certificates:
+
+
+client.crt / client.key
+=======================
+
+Conan support client TLS certificates. Create a *client.crt* with the client certificate in the
```

Comment:
I think this might not be correct. This is now an opt-in via ``core.net.http:client_cert`` conf, so these files no longer belong to the "configuration" set